### PR TITLE
Feature/map view

### PR DESCRIPTION
**Pull request by jpirazusta**
Google map added to home screen.

Simulator Screen Shot - iPhone 11 - 2020-04-13 at 14 57 23

### CODE REVIEW

**Review by ManuViola77 — APPROVED** [review]
Great job making google map work 👏

**Review by acollazomayer — APPROVED** [review]

File: src/screens/MainScreen/styles.js [hunk 1]
```diff
@@ -3,10 +3,16 @@ import { StyleSheet } from 'react-native';
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
     backgroundColor: '#F5FCFF',
```
Comment (elizabethlofredo) [inline src/screens/MainScreen/styles.js:6]: use color constants here

File: src/screens/MainScreen/index.js [hunk 2]
```diff
@@ -14,14 +15,23 @@ const MainScreen = () => {
...
-      <Text>Hey{` ${email}` || ''}, you&#39;re logged in!</Text>
-      <Button testID="logout-button" onPress={logoutRequest} title={strings.MAIN_SCREEN.logout} />
+      <MapView
+        provider={PROVIDER_GOOGLE}
+        style={styles.map}
+        region={{
+          latitude: 37.78825,
+          longitude: -122.4324,
+          latitudeDelta: 0.0922,
+          longitudeDelta: 0.0421,
```
Comment (elizabethlofredo) [inline src/screens/MainScreen/index.js:25]: Maybe you can store these variables on a constant ?

**Review by elizabethlofredo — COMMENTED** [review]
Great job!!!

**Pull request merged**